projects/packages/assets/changelog/fix-jsx-runtime-react-19-polyfill

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Could go either way, but I'm inclined to look at adding the polyfill as being enough of "adding functionality" to be a minor version bump.
```suggestion
Significance: minor
```